app/developers/templates/developers/collection_submit_confirmation.html

```diff
@@ -0,0 +1,38 @@
+{% from "govuk_frontend_jinja/components/panel/macro.html" import govukPanel %}
+{% extends "developers/access_grant_funding_base.html" %}
+
+{% block pageTitle %}
+  Collection submitted {{ collection_helper.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% set reference = collection_helper.id | string | truncate(8, True, "") %}
```

Comment:
Even if it's just a mock reference for now, I think I'd rather generate it either on the collection model or the collection_helper, just to get it out of the template 🙏